_episodes/12-cluster.md

```diff
@@ -46,6 +47,13 @@ tasks.
 
 
 ## Logging in
+The first step in using a cluster is to establish a connection from our laptop to the cluster,
+via the internet. We use a program called the Secure SHell or ssh for this. (make sure you have 
+this program in your laptop and refer the *setup* section for more details).
```

Comment:
I suggest linking to the setup page via the `/setup/` permalink here.

Reply:
Forgot the format for linking pages under same repo, was it a relative URL ?

Reply:
I *think* it's `["setup"]({{ page.root }}/setup/)`

Reply:
e7dfebe

Reply:
e7dfebe